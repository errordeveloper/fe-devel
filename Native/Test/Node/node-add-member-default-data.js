/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

FABRIC = require('Fabric').createClient();

var node = FABRIC.DG.createNode("node");
node.addMember("foo","Scalar",3.141);
console.log(Math.round(node.getData("foo", 0)*10000)/10000);
node.setData("foo", 0, 2.718);
console.log(Math.round(node.getData("foo", 0)*10000)/10000);
node.resize(2);
console.log(Math.round(node.getData("foo", 1)*10000)/10000);

FABRIC.close();
