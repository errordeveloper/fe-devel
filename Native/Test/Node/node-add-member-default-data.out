3.141
2.718
3.141
